fluidefi/fluidefi-caspernet-aggregator-tools
# Block finder by timestamp

**Summary of change:**
- Added a new micro-service, block finder by timestamp, which accepts timestamp as an argument and returns the block. 

**List of tasks:** 


**Steps for testing**
- Make sure your local git is up to date
```bash
git pull
```

- Checkout the branch `feature/block-finder-by-timestmap`
```bash
git checkout feature/block-finder-by-timestmap
```

- Run the test across the test-net
```bash
npm run test
```

- Confirm block-finder test has passed.
- Run example.
